hypervisor/src/kvm/mod.rs

```diff
@@ -720,38 +672,42 @@ impl vm::Vm for KvmVm {
             .map_err(|e| vm::HypervisorVmError::SetGsiRouting(e.into()))
     }
 
+    /// Creates a guest physical memory region.
     ///
-    /// Creates a memory region structure that can be used with {create/remove}_user_memory_region
+    /// # Safety
     ///
-    fn make_user_memory_region(
+    /// userspace_addr must point to memory_size bytes of memory
```

Comment:
```suggestion
    /// `userspace_addr` must point to `memory_size` bytes of memory
```

Doc comments should be Markdown.